sys/platform/src/test.rs

```diff
@@ -0,0 +1,84 @@
+use super::*;
+
+#[test]
+fn dump_features() {
+    eprintln!("simd128\t\t{:?}", simd128_support());
+    eprintln!("simd256\t\t{:?}", simd256_support());
+    eprintln!("x25519\t\t{:?}", x25519_support());
+    eprintln!("bmi2 & adx\t{:?}", bmi2_adx_support());
+    eprintln!("aes\t\t{:?}", aes_ni_support());
+    eprintln!("advSimd\t\t{:?}", adv_simd_support());
+    eprintln!("pmull\t\t{:?}", pmull_support());
+    eprintln!("sha256\t\t{:?}", sha256_support());
+}
+
+// XXX[windows]: Something is running into a STATUS_ACCESS_VIOLATION here.
+#[cfg(all(
+    any(target_arch = "x86", target_arch = "x86_64"),
+    not(target_os = "windows")
```

Comment:
I don't think we need this guard. It works for me on Windows.

Reply:
Done!